### PR TITLE
Seperate sections into their own files

### PR DESCRIPTION
**Pull request by norMNfan**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+= Coding Conventions
+
+link:#Overview[Overview] | link:#Decisions[Decisions] | link:https://github.com/konohiroaki/code-review-note/blob/master/naming.adoc[Naming] | link:https://github.com/konohiroaki/code-review-note/blob/master/functions.adoc[Functions] | link:https://github.com/konohiroaki/code-review-note/blob/master/comments.adoc[Comments] | link:https://github.com/konohiroaki/code-review-note/blob/master/learning.adoc[Learning]
```
Comment (konohiroaki) [inline README.adoc]: I think the files should be specified with relative path. I couldn't follow the link in this branch.

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+link:https://google.github.io/styleguide/javaguide.html[Google Java Style Guide]
+
+== Decisions
+
+1. Use foos instead of fooList
+a. Use fooList for interfaces
+
+2. Do not use the this keyword
+
+3. Use foo != null instead of Objects.nonNull(foo)
```
Comment (konohiroaki) [inline README.adoc]: This is actually explained in its javadoc, so it might be redundant.

Comment (norMNfan) [reply README.adoc]: I will remove this one

File: README.adoc (hunk 1 above)
Comment (norMNfan) [reply README.adoc]: I changed this now.

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+== Overview
+
+link:https://google.github.io/styleguide/javaguide.html[Google Java Style Guide]
+
+== Decisions
+
+1. Use foos instead of fooList
+a. Use fooList for interfaces
+
+2. Do not use the this keyword
```
[the remark this answers is not in the document]
Comment (norMNfan) [reply README.adoc]: If you have any good ideas for that please let me know.

File: learning.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
+= Learning
```
[the remark this answers is not in the document]
Comment (norMNfan) [reply learning.adoc]: I will add a brief description for each learning content

**Pull request merged**